File: Sources/BiangBiangUI/Config/Branding.swift
public struct Branding: Sendable {
    public let appName: String
    public let accentColorHex: String
    public let logoAssetName: String
    public let buttonLogoAssetName: String
    public let githubRepo: String // e.g. "veeso/BiangBiang-Hanzi"
    public let supportEmail: String
    public let appStoreId: String
    public let playStoreId: String
    public init(appName: String, accentColorHex: String, logoAssetName: String,
                buttonLogoAssetName: String, githubRepo: String, supportEmail: String,
                appStoreId: String, playStoreId: String)
    {
        self.appName = appName; self.accentColorHex = accentColorHex
        self.logoAssetName = logoAssetName; self.buttonLogoAssetName = buttonLogoAssetName
        self.githubRepo = githubRepo; self.supportEmail = supportEmail
        self.appStoreId = appStoreId; self.playStoreId = playStoreId
    }
}

File: android/biangbiang-ui/src/main/java/dev/veeso/biangbiangui/config/Branding.kt
package dev.veeso.biangbiangui.config

/** App-level branding metadata. Mirrors iOS `Branding`. */
data class Branding(
    val appName: String,
    val accentColorHex: String,
    val logoAssetName: String,
    val buttonLogoAssetName: String,
    /** e.g. "veeso/BiangBiang-Hanzi" */
    val githubRepo: String,
    val supportEmail: String,
    val appStoreId: String,
    val playStoreId: String,
)
